php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2408,6 +2451,124 @@ class NDB_BVL_Instrument extends NDB_Page
     }
 
     /**
+     * Returns a JSON representation of the currently instantiated
+     * instrument capatible with the survey module
+     *
+     * @return string containing valid JSON
+     */
+    function toDirectJSON()
```

Comment:
Why don't you make a new class that extends NDB_BVL_Instrument and put the new code in there instead of adding even more to the NDB_BVL_Instrument class? (Then you can just override the "normal" methods in NDB_BVL_Instrument like getReview and toJSON() instead of having to add new methods to the base class.)